src/NHSD.BuyingCatalogue.Organisations.Api/Repositories/UsersRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using NHSD.BuyingCatalogue.Organisations.Api.Data;
+using NHSD.BuyingCatalogue.Organisations.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Repositories
+{
+    public class UsersRepository : IUsersRepository
+    {
+        private readonly ApplicationDbContext _context;
+
+        public UsersRepository(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<IEnumerable<ApplicationUser>> GetUsersByOrganisationIdAsync(Guid id)
```

Comment:
Although the method name is helpful in this instance the parameter variable 'id' can be a bit vague. Do you think it would be a good idea to be explicit and specify organisationId instead?

Reply:
sure, I can do that